src/components/chat/ChatMessageList.tsx

```
@@ -0,0 +1,119 @@
+          </div>
+
+          {message.type === 'user' && (
+            <div className='bg-border-base rounded-full p-2 h-fit'>
+              <User className='w-5 h-5 text-text-secondary' />
+            </div>
+          )}
+        </div>
+      ))}
+
+      {isLoading && (
+        <div className='flex gap-3' role="status" aria-live="polite" aria-label="답변 생성 중">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)